clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -16,20 +16,19 @@ import caseapp.core.commandparser.CommandParser
   */
 sealed trait ClioCommand
 
-sealed trait GetSchemaCommand extends ClioCommand {
-  def index: TransferIndex
-}
+sealed abstract class GetSchemaCommand[TI <: TransferIndex](
+  val index: TransferIndex
```

Comment:
Shouldn't this be `TI` not `TransferIndex`?

Reply:
Yup, good catch